.github/workflows/ci.yml

```diff
@@ -36,8 +36,8 @@ jobs:
         sudo apt-get install ./pandoc-3.1.4-1-amd64.deb
     - name: Install dev dependencies
       run: |
-        python -m pip install --upgrade pip
-        # remove pygfx from requirements, we install a specific commit of pygfx since both fpl and pygfx are fast evolving
+        python -m pip install --upgrade pip setuptools
+        # remove pygfx from install_requires, we install using pygfx@main
```

Comment:
is this the best thing to do? we often fall behind pygfx `main` means tests could be failing in the event PRs are made before we catch-up

or is this to force us to be caught up always lol

Reply:
yea this way we catch things before they sneak up on us :D